noether_msgs/action/SimulateThickness.action

```diff
@@ -0,0 +1,10 @@
+#goal definition - Input Mesh
```

Comment:
This should probably be thought of as simulating the work function. One might use Noether for sanding, grinding, power washing, applying paint, chemicals, laser paint removal, sand blasting etc. A similar simulation applies to all. The results should be work_complete rather than painted. Or perhaps coverage would be a better term.